hooks/post/30-upgrade.rb

```diff
@@ -31,13 +31,15 @@ def import_subscriptions
 end
 
 def elasticsearch_message
-  return true unless Kafo::Helpers.execute('rpm -q elasticsearch')
-
+  if `rpm -q elasticsearch`.gsub!(/\A"|"\Z/, '') != nil
```

Comment:
@stbenjam sorry about that it was supposed to be rpm -qa like this

  if `rpm -qa elasticsearch`.gsub!(/\A"|"\Z/, '') != nil

hence why i was doing the .gsub to kill off the "" on the string so it truly was nil.

With the new code:

Upgrade Step: elasticsearch_message...
[ INFO 2017-01-30 13:52:29 verbose] Upgrade Step: elasticsearch_message...
[ INFO 2017-01-30 13:52:30 verbose] Elasticsearch is already removed, skipping
Upgrade Step: remove_gutterball...
[ INFO 2017-01-30 13:52:30 verbose] Upgrade Step: remove_gutterball...
[ INFO 2017-01-30 13:52:31 verbose] Gutterball is already removed, skipping
Upgrade completed!

If that is fine let me know and I will clear up the rubocop errors and resubmit